README.md

```diff
@@ -41,34 +91,17 @@ simdjson is easily consumable with a single .h and .cpp file.
    100 results.
    ```
 
-## How It Works
+## About simdjson
 
-simdjson's startling speed is a result of research into the best ways to take advantage of modern superscalar architectures. The biggest factors are using parallel/vector algorithms and SIMD instructions to eliminate branches, reducing data dependency, and careful attention to cache.
+simdjson takes advantage of modern microarchitectures, parallelizing with SIMD vector instructions, reducing branch misprediction, and reducing data dependency to take advantage of each CPU's multiple execution cores.
 
-* A description of the design and implementation of simdjson is in our research article in VLDB journal: Geoff Langdale, Daniel Lemire, [Parsing Gigabytes of JSON per Second](https://arxiv.org/abs/1902.08318), VLDB Journal 28 (6), 2019appear)
+Some people [enjoy reading our paper](https://arxiv.org/abs/1902.08318): A description of the design and implementation of simdjson is in our research article in VLDB journal: Geoff Langdale, Daniel Lemire, [Parsing Gigabytes of JSON per Second](https://arxiv.org/abs/1902.08318), VLDB Journal 28 (6), 2019appear)
 
 We also have an informal [blog post providing some background and context](https://branchfree.org/2019/02/25/paper-parsing-gigabytes-of-json-per-second/).
 
-Some people [enjoy reading our paper](https://arxiv.org/abs/1902.08318):
-
-[<img src="images/halvarflake.png" width="50%">](https://twitter.com/halvarflake/status/1118459536686362625)
-
-## Talks
-
-QCon San Francisco 2019 (best voted talk):
-
-[![simdjson at QCon San Francisco 2019](http://img.youtube.com/vi/wlvKAT7SZIQ/0.jpg)](http://www.youtube.com/watch?v=wlvKAT7SZIQ)
+For the video inclined, [![simdjson at QCon San Francisco 2019](http://img.youtube.com/vi/wlvKAT7SZIQ/0.jpg)](http://www.youtube.com/watch?v=wlvKAT7SZIQ) was the best voted talk.
 
-## Real-world usage
-
-- [Microsoft FishStore](https://github.com/microsoft/FishStore)
-- [Yandex ClickHouse](https://github.com/yandex/ClickHouse)
-- [Clang Build Analyzer](https://github.com/aras-p/ClangBuildAnalyzer)
-- [azul](https://github.com/tudelft3d/azul)
-
-If you are planning to use simdjson in a product, please work from one of our releases.
-
-## Performance results
+### Performance results
 
 simdjson uses three-quarters less instructions than state-of-the-art parser RapidJSON and fifty percent less than sajson. To our knowledge, simdjson is the first fully-validating JSON parser to run at gigabytes per second on commodity processors.
```

Comment:
I guess that this goes back to before this edit, but my impression is that this is not proper English ("simdjson uses three-quarters less instructions than state-of-the-art parser RapidJSON and fifty percent less than sajson.") and it does not start with an upper case letter. What we have been doing is writing "The simdjson library..." to avoid the ugly "Simdjson". Now, we can either try to fix this sentence now or later. Note: I am aware that you did not author this sentence (neither did I). Here is the commit where this sentence was introduced https://github.com/simdjson/simdjson/commit/0213c3b4d688bd04295e144e152e16926b198f71

Reply:
I'll do it. I understand the squeamishness about it. Even if it turns out that it's "proper English" to do this with proper uncapitalized names (not sure), it's going to trip some people up when reading :)